esti/ops/docker-compose-common.yaml

```diff
@@ -0,0 +1,78 @@
+services:
+  lakefs:
+    image: "${REGISTRY:-treeverse}/${REPO:-lakefs}:${TAG:-dev}"
+    command: "${COMMAND:-run}"
+    ports:
+      - "8000:8000"
+    volumes:
+      - lakefs-app:/app:ro
+    environment:
+      - LAKEFS_AUTH_ENCRYPT_SECRET_KEY=some random secret string
+      - LAKEFS_AUTH_UI_CONFIG_RBAC=${LAKEFS_AUTH_UI_CONFIG_RBAC:-none}
+      - LAKEFS_AUTH_API_ENDPOINT=${LAKEFS_AUTH_API_ENDPOINT:-}
+      - LAKEFS_BLOCKSTORE_TYPE=${LAKEFS_BLOCKSTORE_TYPE:-local}
+      - LAKEFS_BLOCKSTORE_LOCAL_PATH=/home/lakefs
+      - LAKEFS_BLOCKSTORE_LOCAL_IMPORT_ENABLED=true
+      - LAKEFS_BLOCKSTORE_LOCAL_ALLOWED_EXTERNAL_PREFIXES=/tmp
+      - LAKEFS_BLOCKSTORE_S3_CREDENTIALS_ACCESS_KEY_ID=${AWS_ACCESS_KEY_ID}
+      - LAKEFS_BLOCKSTORE_S3_CREDENTIALS_SECRET_ACCESS_KEY=${AWS_SECRET_ACCESS_KEY}
+      - LAKEFS_LOGGING_LEVEL=DEBUG
+      - LAKEFS_BLOCKSTORE_GS_CREDENTIALS_JSON
+      - LAKEFS_STATS_ENABLED
+      - LAKEFS_USAGE_REPORT_ENABLED=false
+      - AZURE_CLIENT_ID
+      - AZURE_CLIENT_SECRET
+      - AZURE_TENANT_ID
+      - LAKEFSACTION_VAR=this_is_actions_var
+    extra_hosts:
+      - "host.docker.internal:host-gateway"
+
+  esti:
+    image: "golang:1.23-alpine"
+    links:
+      - lakefs:s3.local.lakefs.io
+      - lakefs:testmultipartupload.s3.local.lakefs.io
+      - lakefs:testmultipartuploadabort.s3.local.lakefs.io
+      - lakefs:testdeleteobjects.s3.local.lakefs.io
+      - lakefs:testmigrate-testpremigratemultipart.s3.local.lakefs.io
+      - lakefs:migrate.s3.local.lakefs.io
+    environment:
+      - CGO_ENABLED=0
+      - AWS_ACCESS_KEY_ID
+      - AWS_SECRET_ACCESS_KEY
+      - AWS_REGION=us-east-1
+      - ESTI_STORAGE_NAMESPACE
+      - ESTI_BLOCKSTORE_TYPE
+      - ESTI_AWS_ACCESS_KEY_ID
+      - ESTI_SETUP_LAKEFS
+      - ESTI_AWS_SECRET_ACCESS_KEY
+      - ESTI_ENDPOINT_URL=http://lakefs:8000
+      - ESTI_BINARIES_DIR=/app
+      - ESTI_GOTEST_FLAGS
+      - ESTI_FLAGS
+      - ESTI_FORCE_PATH_STYLE
+      - ESTI_AZURE_STORAGE_ACCOUNT
+      - ESTI_AZURE_STORAGE_ACCESS_KEY
```

Comment:
Any environment variable here that is not set is not needed in the common file. Either set them with a default that can be used by the extending configurations or remove them from here and define it there